infor-design/enterprise-wc-examples
# 1411 - Angular Demos: added example multiselect demonstrating ngFor use

**Explain the details for making this change. What existing problem does the pull request solve?**

Added example IDS Multiselect to Angular examples to test use with `ngFor` directive.

**Related github/jira issue (required)**:


Closes https://github.com/infor-design/enterprise-wc/issues/1411

**Steps necessary to review your pull request (required)**:


1. Check out this branch
2. Run: `rm -fr .angular/cache`
3. Then run: `npm run build && npm run start` 
6. Go to: http://localhost:4200/ids-multiselect/example
7. See 4th example called `Test Multiselect (using ngFor)`
8. Go to: http://localhost:4200/ids-forms/example
9. Open your browser's developer console to see updated values for the `ids-multiselect` input.
